Cargo.lock

```diff
@@ -12463,9 +12463,9 @@ dependencies = [
 
 [[package]]
 name = "tokio"
-version = "1.43.1"
+version = "1.44.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "492a604e2fd7f814268a378409e6c92b5525d747d10db9a229723f55a417958c"
+checksum = "e6b88822cbe49de4185e3a4cbf8321dd487cf5fe0c5c65695fef6346371e9c48"
 dependencies = [
```

Comment:
Hmm... I think we normally try to stay on the [long-term releases of tokio](https://github.com/tokio-rs/tokio?tab=readme-ov-file#bug-patching-policy). Is there a new requirement in qorb driving this? If not it might be nice to set the dep in qorb's Cargo.toml to 1.43 and republish it.

Reply:
Sure, happy to downgrade, will do in https://github.com/oxidecomputer/qorb/pull/106

Reply:
Updated

Reply:
hrm, hang on, still seeing 1.44 here...

Reply:
Okay, *now* it should be resolved